.bit/responses/1.8-Week 1 Step 8.md

```diff
@@ -102,12 +102,29 @@ The next one actually installs the necessary packages with code, `morse-code-con
 
   **Tip**: Try reading the [documentation](https://www.npmjs.com/package/morse-code-converter) first.
 
+1. First require the npm package at the top of your code.
 ```js
 const morse = require("morse-code-converter");
- 
-const code = morse.textToMorse('Hey how are you?'); // .... . -.--   .... --- .--   .- .-. .   -.-- --- ..- ..-..
-const text = morse.morseToText(code); // HEY HOW ARE YOU?
 ```
+2. Query the url for the parameter `plaintext`, and store it in a variable. 
+3. Create a variable named code, but set it to `undefined` for right now, because we are not sure if `plaintext` contains a value or not (we can't translate nothing 🤔)
+4. To check if the user passed in nothing for plaintext, we need to use an `if-else` [conditional](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else).
+5. We first check if the user did not pass in a parameter of `plaintext` at all, or if `plaintext` has no value. In which case, we will tell them to enter some text. 
+
+```js
+if (typeof plaintext === 'undefined' || plaintext === "") {
+        code = "Please enter some text to  convert!"
+}
+```
+> 💡 the `||` means `or` in JavaScript. Either the left side can be `true`, or the right side can be `true`, and the code inside the `if` will run!
+
+6. Now that we checked if the user has entered nothing, we can add code that will execute when the user has entered something for `plaintext`. Add this after the `if`.
```

Comment:
I am not sure this is perfectly clear as it is not directly after the if. The next line also does not have the extra bracket needed to make it work. Not sure how to make it clearer without the entire code sample.